.changeset/nice-walls-reply.md

```diff
@@ -0,0 +1,14 @@
+---
+'@backstage/plugin-catalog-common': minor
+---
+
+Mark permission-related exports as alpha. This means that the exports below should now be imported from `@backstage/plugin-catalog-common/alpha` instead of `@backstage/plugin-catalog-common`.
```

Comment:
```suggestion
**Breaking**: Mark permission-related exports as alpha. This means that the exports below should now be imported from `@backstage/plugin-catalog-common/alpha` instead of `@backstage/plugin-catalog-common`.
```

Reply:
Also sneaking in the same fix for the catalog-backend changeset in 5c59257300.